### ISSUE TITLE
Extension - Mostly not working for me

### ISSUE THREAD

**Issue by dkirrane** [hashicorp/vscode-terraform#302]
Hi, I'm hitting a number of issues with this plugin.

- `Ctrl+Click` Document linking not working. Link doesn't appear:
![image](https://user-images.githubusercontent.com/1280116/78989721-51d65580-7b2c-11ea-8b57-e76bbcde752b.png)

- I tried the `Terraform: reindex terraform files` command but get this in `View > Output > Terraform`
```
2020-04-09 21:07:56:244 [INFO ] runner: Found Terraform binary C:\ProgramData\chocolatey\bin\terraform with version 0.12.24
2020-04-09 21:07:56:440 [INFO ] file-system-watcher: Crawling workspace for terraform files...
2020-04-09 21:07:56:735 [INFO ] file-system-watcher: Index not generated for: file:///c%3A/DEV/Azure/azure-terraform/aks-cluster/main.tf
```
- Terraform Explorer doesn't show anything & picks up the vscode `.history` backup directory

![image](https://user-images.githubusercontent.com/1280116/78989580-e8eedd80-7b2b-11ea-9323-4aca4ebcf49e.png)

**Comment by mattburgess** [comment]
Here's a number of my frustrations borne out by this ticket:

1. I have 0.4.0 of the extension installed, though I've no idea where that's come from/what it's been built from https://github.com/mauve/vscode-terraform/releases only goes as far as 0.3.12. The latest commit on this repo is from back in October 2019.
2. Having installed 0.4.0, it prompted me to download the language server which I duly did (selected 0.10.0 as it was latest at the time). Is that version compatible with this extension version? Who knows? Why can't it work out which version to download (defaulting to latest would seem sensible)
3. Encountering the same issues as the OP here, I googled and found out that `indexing` is a thing, so switched that on, only to be told that those settings are incompatible with the language server. That's fine, but then I went hunting for what configuration settings I might be able to give to the language server to make it do the right thing and here's the fun part!
https://github.com/juliosueiras/terraform-lsp#supported-editors links to https://github.com/juliosueiras/terraform-lsp/blob/master/docs/editors/vscode.md which says **DEPRECATED** and links to https://github.com/juliosueiras/vscode-languageclient-terraform which also says **DEPRECATED** and links me full circle back to this repo, which doesn't appear to have any docs for configuring the LSP integration.

So, I have a few questions:

1. Where's the source repository for v1.4.0 of the Terraform plugin?
2. Where's the source repository for v0.10.0 of the language server?
3. Are the features for code navigation and doc launching expected to work out of the box, and if not, where are the docs for what settings need to be changed to make those core features work?

**Comment by cyrus-mc** [comment]
Running into same issues. Have had no success in resolving.

**Comment by paultyng** [comment]
We just released [v2.0.0-rc.1](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) of the extension. The main features include:

* Added syntax support for 0.12
* Added [terraform-ls](https://github.com/hashicorp/terraform-ls) usage by default (currently on 0.3.0, which offers basic provider code completion)

You can find additional information and specifics in the [release notes](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) and [CHANGELOG](https://github.com/hashicorp/vscode-terraform/blob/master/CHANGELOG.md).

With this release we expect that many of the prior issues and PRs are no longer relevant or have been addressed, and are therefore being closed. If you feel the action taken on an issue or PR is in error, please comment as such and we can figure out the appropriate way to address it.

We plan to add the final 2.0.0 release to the marketplace soon, but are actively seeking your feedback now on the release candidates. You can download the .vsix from the releases page and [manually install it in VS Code](https://code.visualstudio.com/docs/editor/extension-gallery#_install-from-a-vsix) to try it out.
